### ISSUE TITLE
Can’t create pull request

### ISSUE THREAD

**Issue by craigfairhurst8** [AlmaLinux/mirrors#1125]
For some reason, I can’t for the life of me figure out why I can’t create a pull request. Anyways, please see here for what I could do: https://github.com/AlmaLinux/mirrors/commit/aa3607ee79a595480f3c5cb3c5e42474277ef31a

Apologies for the confusion.

**Comment by craigfairhurst8** [comment]
Still can’t. Tried from desktop. Why!

**Comment by Sparkey36079** [comment]
Yeap I have the same problems not to sure how to correct this error? 
Are you getting this same error?
Errors during downloading metadata for repository 'appstream':
  - Curl error (23): Failed writing received data to disk/application for http://cofractal-sea.mm.fcix.net/almalinux/9.5/AppStream/x86_64/os/repodata/09830ba495bb923697bda781fcd3accf985248d47d8f4983b13c094cd56ef960-primary.xml.gz [Failure writing output to destination]
  - Curl error (23): Failed writing received data to disk/application for http://cofractal-sea.mm.fcix.net/almalinux/9.5/AppStream/x86_64/os/repodata/25670428f6226304883ca2ab9c8aceb9b9da68f4646ec7d51c47a7408d4bcf1f-updateinfo.xml.gz [Failure writing output to destination]
  - Curl error (23): Failed writing received data to disk/application for http://cofractal-sea.mm.fcix.net/almalinux/9.5/AppStream/x86_64/os/repodata/1c7216da8e9df5b1c58096ab6451748034c3149badf9d726595d26da509deb9f-filelists.xml.gz [Failure writing output to destination]
This goes on throughout all the repo's.

**Comment by codyro** [comment]
@craigfairhurst8 What issue did you have when attempting to make a PR? I can help walk you through the process so we can get this fixed up.

**Comment by codyro** [comment]
The button will be unclickable until you show which changes you want to merge where. If you look at the `Compare` portion up top, you will find your local changes, which you will want to merge into our `master` branch. Once that is done, you will see the diff with your changes, and the button will be clickable.

**Comment by craigfairhurst8** [comment]
I haven't made any changes, I imagined that the pull request would generate the files for me to edit or add to.

All I want to do is include AlmaLinux mirror info.

**Comment by craigfairhurst8** [comment]
I've since pulled via git, made changes, how do I send them back?

thanks for the help.

**Comment by craigfairhurst8** [comment]
![Image](https://github.com/user-attachments/assets/7d7e91f4-612c-42af-8d84-ff92f72967f0)
